### PR TITLE
add geohash2 recipe

### PR DESCRIPTION
**Pull request by david-gang**
This is my first PR :-)

I add the geohash2 library which should be managed by conda.

### CODE REVIEW

File: recipes/geohash2/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,661 @@
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```
Comment (synapticarbors) [inline recipes/geohash2/LICENSE.txt]: It looks like the license file is already distributed with the source code, so I think you can remove this recipe-local copy.

Comment (david-gang) [reply recipes/geohash2/LICENSE.txt]: Ok. Going to delete the license.txt

Comment (david-gang) [reply recipes/geohash2/LICENSE.txt]: Deleted the license.txt. Please let me know if everything is ok

**Comment by synapticarbors** [comment]
@david-gang -- Thanks for your contribution to conda-forge. A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Comment by david-gang** [comment]
Hi @synapticarbors,
Now it comes to my mind that the geohash2 package is built specifically for python3.
Do i need to add something in the recipe to indicate this?
Thanks

**Comment by synapticarbors** [comment]
Does it explicitly not run on python2?

**Comment by david-gang** [comment]
I tried to check it and can get something weird:
conda install geohash2
Fetching package metadata .............
Solving package specifications: .
Warning: 2 possible package resolutions (only showing differing packages):
  - conda-forge::python-3.6.2-0, conda-forge::zlib-1.2.11-0
  - conda-forge::python-3.6.3-0, conda-forge::zlib-1.2.8-3

Package plan for installation in environment /home/david/anaconda3/envs/test:

The following NEW packages will be INSTALLED:

    ca-certificates: 2017.7.27.1-0      conda-forge
    certifi:         2017.7.27.1-py36_0 conda-forge
    geohash2:        1.1-py_0           conda-forge
    ncurses:         5.9-10             conda-forge
    openssl:         1.0.2l-0           conda-forge
    pip:             9.0.1-py36_0       conda-forge
    python:          3.6.2-0            conda-forge
    readline:        6.2-0              conda-forge
    setuptools:      36.6.0-py36_1      conda-forge
    sqlite:          3.13.0-1           conda-forge
    tk:              8.5.19-2           conda-forge
    wheel:           0.30.0-py_1        conda-forge
    xz:              5.2.3-0            conda-forge
    zlib:            1.2.11-0           conda-forge

Proceed ([y]/n)? 


Why does it want to install python 3.6.2?

**Comment by synapticarbors** [comment]
In a python2 environment, I just ran:

```
$ conda create -n geohash2 geohash2
```

And then in the python repl:

```
>>> import geohash2
>>> from __future__ import print_function
>>> print ('Geohash for 42.6, -5.6:', geohash2.encode(42.6, -5.6, precision=5))
Geohash for 42.6, -5.6: ezs42
```

so it looks to run ok in python 2.7. I'm not sure what is going on with the zlib issue that you're showing. It looks like they are based on two separate python versions 3.6.2 and 3.6.3

**Comment by david-gang** [comment]
should i uninstall and reinstall python?

**Pull request merged**